packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+
+import 'box.dart';
+import 'object.dart';
+import 'shifted_box.dart';
+
+/// A render object that can render child size change animations.
```

Comment:
I'd either flesh this out to a complete description of the class or remove it.  We have a tool that tells us which dartdocs we're missing.  If we have no docs, the tool will flag this location but if we have this sort of minimal doc, then we're likely to miss improving it.